Add pull to refresh without id

EsteveAguilera opened the pull request:
Add a `refresh()` method without the need to add an `id` as parameter

Fixes the #133 with special thanks to @Sloy for the hint! 😜

Sloy commented:
You're on fire! 🔥 
Let me review everything from work tomorrow :)

If you're curious, you can take a look at https://github.com/SchibstedSpain/Barista/pull/146 to see a cool way to do the id magic by using kotlin's default parameter values.

EsteveAguilera commented:
You're right @Sloy! 😚 

I've converted the class to Kotlin and used the default parameter value

I've also renamed the `swipeRefreshId` parameter to `id`... This seems to be the convention on the rest of the project

Take the time you need to review all my changes! 
These are my first contributions, and I'm still not too familiar with the project...

EsteveAguilera commented:
🕺
